src/asset-management/AssetManagedPair.sol

```diff
@@ -127,12 +127,5 @@ abstract contract AssetManagedPair is Pair, IAssetManagedPair {
 
             IERC20(token1).transferFrom(address(assetManager), address(this), lDelta);
         }
-
-        _update(
-            _totalToken0(),
-            _totalToken1(),
-            reserve0,
-            reserve1
-        );
```

Comment:
removed this as there is no need it for it. The reserves would still be the same before and after the transfer to AM. It's only the managed amounts that change

Reply:
Unless PNL has been accrued but that should be handled separately, so this seems good